tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -988,9 +986,16 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                               default=())
     # TODO: isn't it a weights precision type
     common_group.add_argument('--data_type',
-                              help=mo_convert_params['data_type'].description,
+                              help='[DEPRECATED] Data type for all intermediate tensors and weights. '
```

Comment:
This description is partially incorrect: all intermediate data is still in the original precision, not obligatory in FP16. `--data_type=FP16` works identically as `--compress_to_fp16=True`.

Reply:
Suggested new description in the comment above. Please check.